Framework/Kernel/src/ErrorReporter.cpp

```diff
@@ -101,6 +101,8 @@ std::string ErrorReporter::generateErrorMessage() {
 
   message["exitCode"] = m_exitCode;
 
+  message["textBox"] = m_textbox;
```

Comment:
We probably want this inside the m_share check below so that it is shared under the same conditions as name and email address